lib/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 plugins {
     id("local.base")
     id("local.publish")
+    id("org.jetbrains.dokka") version "1.8.10"
```

Comment:
This should be a part of other ticket that will also confogire more stuff https://lemonappdev.atlassian.net/jira/software/projects/KON/boards/2?selectedIssue=KON-34&text=dok